### PR TITLE
332 inplace sel

### PR DESCRIPTION
**Pull request by dfsp-spirit**
Author Guidelines
-----------------
- [x] Is the change set **< 600 lines**?
- [x] Was the code checked for memory leaks/performance bottlenecks?
- [x] Is the code running locally **and** on the ESI cluster?
- [x] Is the code running on all supported platforms?

Reviewer Checklist
------------------
- [ ] Are testing routines present?
- [ ] Do **parallel** loops have a set length and correct termination conditions?
- [ ] Do objects in the global package namespace perform proper parsing of their input? 
- [ ] Do code-blocks provide novel functionality, i.e., no re-factoring using builtin/external packages possible?
- [ ] Code layout
  - [ ] Is the code PEP8 compliant?
  - [ ] Does the code adhere to agreed-upon naming conventions?
  - [ ] Are keywords clearly named and easy to understand?
  - [ ] No commented-out code?
- [ ] Are all docstrings complete and accurate?
- [ ] Is the CHANGELOG.md up to date?

### CODE REVIEW

File: syncopy/tests/test_specest.py [hunk 1]
```diff
@@ -157,6 +158,11 @@ def test_cut_selections(self, fulltests):
             self.sigdataSelections.pop(random.choice([-1, 1]))
             self.artdataSelections.pop(random.choice([-1, 1]))
 
+    @staticmethod
+    def get_adata():
```
Comment (tensionhead) [inline syncopy/tests/test_specest.py:162]: That's a good way to coherently re-initialize data objects :+1: when needed

File: syncopy/tests/test_specest.py [hunk 2]
```diff
@@ -586,7 +599,7 @@ def test_tf_toi(self):
                 cfg.t_ftimwin = winsize
                 for toi in toiVals:
                     cfg.toi = toi
-                    tfSpec = freqanalysis(cfg, self.tfData)
+                    tfSpec = freqanalysis(cfg, TestMTMConvol.get_tfdata_mtmconvol())
```
Comment (tensionhead) [inline syncopy/tests/test_specest.py:602]: However, do we really need to repopulate a virgin object every time here?

**Review by tensionhead — COMMENTED** [review]
Nice and clean :+1: Just wondering if all the re-initializations could bog the tests down? But maybe actually on disc nothing is really happening, hence it is super cheap?!

File: syncopy/tests/test_specest.py (hunk 2 above)
Comment (dfsp-spirit) [reply syncopy/tests/test_specest.py]: I have changed this, the data is now created before the loop and re-used within loop iterations.

Comment (tensionhead) [reply syncopy/tests/test_specest.py]: perfect, thx :+1:

**Pull request merged**